test/test_scanner.py

```diff
@@ -720,6 +734,12 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "png",
                         "1.6.36",
                     ),
+                    (
+                        "https://kojipkgs.fedoraproject.org/packages/ppp/2.4.6/6.fc21/x86_64/",
+                        "ppp-2.4.6-6.fc21.x86_64.rpm",
+                        "ppp",
+                        "2.4.6",
```

Comment:
And when you update this, please change one of the tests to not be version 2.4.6 (again, avoiding that bug we have in the test code)